modules/apps/password-policies-admin/password-policies-admin-web/src/main/resources/META-INF/resources/js/ManagementToolbarDefaultPropsTransformer.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {createActionURL} from 'frontend-js-web';
+
+const ACTIONS = {
+	deletePasswordPolicies(portletNamespace, basePortletURL) {
+		if (
+			confirm(
+				Liferay.Language.get('are-you-sure-you-want-to-delete-this')
+			)
+		) {
+			const form = document.getElementById(`${portletNamespace}fm`);
```

Comment:
We can use `document[``${portletNamespace}fm``];` without `if (form)`

Reply:
I'll remove the `if` as we discussed it (this is something I forgot to clean up)

I must admit I find `document[`${portletNamespace}fm`];` to look ugly, but since we agreed on that I will change it.

Reply:
It's not only SF, it is faster as we don't need to make a DOM query. `<aui:form>` sets `document.formId`.

Reply:
I get your point, I was just "opinionating" (is that even a word :rofl:) - no worries